docs/usage.md

```diff
@@ -152,6 +153,7 @@ The (uncompressed) database paths (`db_path`) for each tool are expected to cont
 - [**MetaPhlAn**:](usage/tutorials.md#metaphlan-custom-database) output of with `metaphlan --install` or downloaded from links on the [MetaPhlAn wiki](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-4#customizing-the-database).
 - [**mOTUs**:](usage/tutorials.md#motus-custom-database) the directory `db_mOTU/` that is downloaded via `motus downloadDB`.
 - [**ganon**:](usage/tutorials.md#ganon-custom-database) output of `ganon build` or `ganon build-custom`.
+- [**kmcp**:](usage/tutorials.md#kmcp-custom-database) output of `kmcp compute` and `kmcp index`.
```

Comment:
Maybe specify if this is two outputs in one directory? or Maybe if compute output goes into index (or vice versa) just use the final step that builds the database that goes into search

Reply:
Since both` kmcp compute `and `kmcp index` are needed to build the database that goes into search wouldn't it be useful to name it here?